cli/command/load_balancer_test.go

```diff
@@ -6,41 +6,323 @@ import (
 	"github.com/quintilesims/layer0/common/models"
 )
 
-func TestCreateLoadBalancer(t *testing.T) {
+func TestAddPortToLoadBalancer(t *testing.T) {
 	base, ctrl := newTestCommand(t)
 	defer ctrl.Finish()
 
-	ports := []models.Port{}
+	base.Resolver.EXPECT().
+		Resolve("load_balancer", "lname").
+		Return([]string{"lid"}, nil)
+
+	ports := []models.Port{
+		models.Port{
+			ContainerPort: 80,
+			HostPort:      80,
+			Protocol:      "tcp",
+		},
+	}
 	healthCheck := models.HealthCheck{}
 
+	base.Client.EXPECT().
+		ReadLoadBalancer("lid").
+		Return(&models.LoadBalancer{
```

Comment:
[nit] this is difficult to read; we should probably put the load balancer model into its own variable.

Reply:
:white_check_mark: